### PR TITLE
Add footer and contact form

### PR DESCRIPTION
**Pull request by Oklukeok**
Added footer section and contact form to the project.
Updated the README file with the project news.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -259,6 +260,36 @@ <h4>Skills</h4>
...
+              <input type="email" id="mail" name="user_email" required placeholder="lucasabermudez@gmail.com">
+          </li>
+          <li>
+              <label for="msg"></label>
+              <textarea id="msg" name="user_message" maxlength="500" cols="30" rows="10" required placeholder="Write your message here"></textarea>
+        </ul>
+      <button type="submit" id="sub">Get in touch</button>
+
+      </form>
+    </footer>
```
Comment (prolajumokeoni) [inline index.html:292]: Figma Design          |  Your design     
:-------------------------:|:-------------------------:
Screenshot_181  |  Screenshot_420


- [ ] Kindly remove the background image add to CSS in the footer, you can add a `background-color` so it takes the entire width 👍🏿 
- [ ] KIndly make your footer and form design look like the Figma design, please use the right padding and margin 👍🏿

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @Oklukeok, 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Git flow was correctly used
✔️  Good PR title and Description



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -259,6 +260,37 @@ <h4>Skills</h4>
...
+              <input type="text" id="name" name="user_name" required placeholder="Lucas Bermudez" maxlength="30">
+          </li>
+          <li>
+              <label for="mail"></label>
+              <input type="email" id="mail" name="user_email" required placeholder="lucasabermudez@gmail.com">
+          </li>
+          <li>
+              <label for="msg"></label>
+              <textarea id="msg" name="user_message" maxlength="500" cols="30" rows="10" required placeholder="Write your message here"></textarea>
+        </ul>
```
Comment (vickymarz) [inline index.html:288]: [OPTIONAL]

- Kindly add the appropriate padding to the input's placeholders to match the Figma design chosen 👍🏽 
Your Design 👇🏽 
image

FIgma Design 👇🏽 

image

**Review by vickymarz — APPROVED** [review]
Hi @Oklukeok , 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**